core/src/databases.rs

```diff
@@ -221,7 +221,13 @@ impl MongoDatabases {
             })
             .unwrap()
             .iter()
-            .filter(|&db_name| !db_name.is_empty() && !db_name.eq("admin"))
+            .filter(|&db_name| {
```

Comment:
Since this list of databases and filter is in a couple places, can we add a helper function to filter out the databases we don't want?